### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by NataliaLKB**
## What does this change?

Adding a code of conduct to our repo

cc @guardian/guardian-frontend-team 

## What is the value of this and can you measure success?
Explicitly stating what kind of behaviour we agree and disagree with

### CODE REVIEW

**Review by SiAdcock — APPROVED** [review]
What's not to love? Great stuff! ❤️

**Review by nicl — APPROVED** [review]

**Review by shtukas — APPROVED** [review]

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a professional setting
```
Comment (JonNorman) [inline CODE_OF_CONDUCT.md]: Nitpick here, but do we need this line, since it is just a list of examples and thus not attempting to be exhaustive?

Comment (shtukas) [reply CODE_OF_CONDUCT.md]: Maybe it should simply say "Including but not limited to"

Comment (NataliaLKB) [reply CODE_OF_CONDUCT.md]: I think its implied in the heading `Examples of unacceptable behavior by participants include:`
I read that as not limited to

**Review by philmcmahon — APPROVED** [review]

**Pull request merged**